packages/world/ts/config/v2/world.ts

```diff
@@ -72,6 +73,30 @@ export type resolveWorldConfig<input> = evaluate<
 >;
 
 export function resolveWorldConfig<const input>(input: validateWorldConfig<input>): resolveWorldConfig<input> {
-  // TODO: runtime implementation
-  return {} as never;
+  const namespaces = get(input, "namespaces") ?? {};
+  const scope = extendedScope(input);
+
+  const namespacedTables = Object.fromEntries(
+    Object.entries(namespaces)
+      .map(([namespaceKey, namespace]) => {
+        const tables = get(namespace, "tables") ?? {};
+        return Object.entries(tables).map(([tableKey, table]) => [`${namespaceKey}__${tableKey}`, table]);
```

Comment:
not sure if we wanna use this here but we have `resourceLabel` in `@latticexyz/common`